.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: node_js
 node_js:
-  - "0.10"
```

Comment:
I've removed old node versions from travis. It now only tests against 4.x and 6.x